cli/src/base.ts

```diff
@@ -43,12 +43,12 @@ export abstract class BaseCommand extends Command {
   searchPlaces = [`.${moduleName}rc`, `.${moduleName}rc.json`, 'package.json'];
 
   static databaseURLFlag = Flags.string({
-    name: 'databaseurl',
+    char: 'u',
```

Comment:
Why not `d`?

Reply:
Because it's a **U**rl and `-d` is used frequently as a _delete_ flag. E.g. `git branch -d to-be-deleted`.

Reply:
I get that `u` means `url` (maybe?), but since it starts with `d`, I expect the shorthand to also be `d`. Even the help page looks a bit more confusing with `-u`:

<img width="251" alt="image" src="https://user-images.githubusercontent.com/9947422/176150028-d81eddbe-400a-4fa7-84ad-05d4df64c385.png">

Reply:
Right, `-d` to delete makes sense. I would still prefer to go with `-d` for now and iterate, possibly changing it to `-u` based on feedback.

Reply:
> git, -D deletes instead of -d

With git `-d` deletes as well. LIke in the example I left above.

Reply:
Right, I'm aware. I would still prefer to go with `-d` for now and iterate, possibly changing it to `-u` based on real-world developer feedback. Can we do this?

Reply:
I prefer not having a shorthand than having a shorthand that can potentially change. I don't want to break users workflows.

Reply:
Would a two char short version, like `-db` be possible? I've noticed they do this in npm, for example: `[-ws|--workspaces]`

Reply:
I'm afraid that's not possible. Oclif asks for a single char.

![image](https://user-images.githubusercontent.com/50486/176159909-198d33a9-7f95-4cd4-bbfc-7d96f6c37271.png)

We could change `databaseURL` to `db`, but not sure if that's clear enough.

Reply:
There's an `enhacement` issue for aliases: https://github.com/oclif/oclif/issues/438

What about duplicating the flag and reading both (databaseURL and db)?

I kind of agree it's a long one

Reply:
Sure, `--db` is much clearer than `-u` and also shorter than `--databaseURL`, which will probably be confused with `--databaseUrl` quite often as well.

Reply:
I'm going to remove the shorthand and will open a different PR for this specific flag.

Reply:
New PR https://github.com/xataio/client-ts/pull/368